src/main/external-resources/renderers/Sony-PlayStation3.conf

```diff
@@ -40,14 +40,15 @@ Supported = f:mp4        v:mp4|h264               a:aac                m:video/m
 Supported = f:wmv        v:wmv|vc1                a:wma                m:video/x-ms-wmv           n:2
 
 # Supported audio formats:
+Supported = f:aac     a:(?!alac).+   n:2             m:audio/x-m4a
+Supported = f:atrac                  n:2             m:audio/ATRAC-X
+Supported = f:lpcm    a:lpcm         n:6   s:48000   m:audio/L16
+Supported = f:m4a				     n:2   s:48000   m:audio/x-m4a
```

Comment:
@Nadahar Yeah, i know as i get myself this kind of issues.

Since few weeks Github advertise, and i verified it, that tabulation was not possible anymore in their interface.